src/ServiceBus.AttachmentPlugin/AzureStorageAttachmentConfiguration.cs

```diff
@@ -10,16 +10,23 @@ public class AzureStorageAttachmentConfiguration
         /// <param name="connectionString"></param>
         /// <param name="containerName"></param>
         /// <param name="messagePropertyToIdentifyAttachmentBlob"></param>
+        /// <param name="sasUri"></param>
```

Comment:
Public property xml doc shouldn't be empty. Would you want to see a no-description doco? 😉